pkg/router/functionHandler.go

```diff
@@ -660,7 +660,7 @@ func (fh functionHandler) getServiceEntryFromExecutor(ctx context.Context) (serv
 	// parse the address into url
 	svcURL, err := url.Parse(fmt.Sprintf("http://%v", service))
 	if err != nil {
-		logger.Error("error parsing service url",
+		logger.Info("error parsing service url",
```

Comment:
I don't think these need to be changed, `logger.Error` logs should already show up in Axiom.

Only `Debug` and lower (e.g. `Trace`) are disabled atm. `Info` and above (e.g. `Warn`, `Error`, etc.) should already be enabled.